.jenkins

```diff
@@ -84,6 +84,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Release \
+                                -DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*" \
```

Comment:
Did you specify `-warnings-as-errors` here on purpose instead of
```
WarningsAsErrors: '*'
```
in the `.clang-tidy` file?

Reply:
Yes.  I think it it is up to the developer if he wants to interpret warnings as errors.